docs/dom-testing-library/api-events.mdx

```diff
@@ -96,6 +96,9 @@ fireEvent.keyDown(domNode, {key: 'Enter', code: 'Enter', charCode: 13})
 fireEvent.keyDown(domNode, {key: 'A', code: 'KeyA'})
 ```
 
+> :warning: Be careful to provide the `charCode` property in case of firing the
+> `keyPress` event. Otherwise, it won't be fired.
+
```

Comment:
This isn't true. The native `keypress` event will be dispatched. React just won't call any `onKeyPress` handlers.
See https://github.com/testing-library/user-event/issues/766

Reply:
Thanks for the response. 
Does it make sense to rephrase that namely the `React` does not fire the event callback?

Reply:
Yes, maybe something along...
```suggestion
> :warning: `React` doesn't dispatch a `SyntheticEvent` for a native `keypress` event without `event.charCode === 13 || event.charCode >= 32`.
> I.e. `onKeyPress` handlers will only be called if you include an appropriate `charCode` property.
```

Reply:
Note, that this is framework-specific and almost all use cases for this should use `user-event` instead.